core/domain/classifier_domain_test.py

```diff
@@ -161,3 +161,111 @@ def test_validation(self):
         with self.assertRaisesRegexp(utils.ValidationError, (
             'Expected classifier_data to be a dict')):
             classifier.validate()
+
+
+class TrainClassifierJobDomainTests(test_utils.GenericTestBase):
```

Comment:
Are we not renaming this to ClassifierTrainingJobDomainTests? (Not that it makes much difference, but might look cleaner?)